meta-xt-prod-devel-rcar-control/recipes-guests/domd/files/domu-generic-h3-4x2g.cfg

```diff
@@ -0,0 +1,55 @@
+# =====================================================================
+# DomU guest configuration
```

Comment:
Please remove this domU.cfg from domD folder.
DomU config is provided by DomU target.